### ISSUE TITLE
Fluids

### ISSUE THREAD

**Issue by ApexModder** [ApexStudios-Dev/Registrator#3]

**Comment by ApexModder** [comment]
Registrate itself has support for custom fluid types, but I am waiting on the [fluid overhaul PR ](https://github.com/MinecraftForge/MinecraftForge/pull/7992)to be merged into Forge before working working on this
